src/giddyup.erl

```diff
@@ -27,6 +27,7 @@
 get_suite(Platform) ->
```

Comment:
Don't know how much this would give us if it were a gen_server.  We'd still end up blocking.  The problem is, we **have** to wait for GiddyUp to finish uploading all of the log files before we can clean up the node and go on.

Easy to pass in the config parameters to giddyup:get_suite/1 if you'd like.